src/data-mgt/python/batch-jobs/airqo/airqo-bigquery.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Hi @NoahNsimbe , please remove this file from this pull request, it has compromised the security credentials.
I am about to disable this service account right away. Thanks.

Reply:
@Baalmart , please disable the service account

Reply:
DONE. already disabled **noah-data-engineering@airqo-250220.iam.gserviceaccount.com**